src/parser/jobs/vpr/modules/snilk.tsx

```diff
@@ -91,7 +88,7 @@ export class snilk extends CoreGauge {
 				Avoid letting your Serpent's Gauge overcap - the wasted resources may cost you uses of your <DataLink action="REAWAKEN"/> burst.
 			</Trans>,
 			why: <Trans id="vpr.gauge.suggestions.overcap.why">
-				{this.snilkGauge.overCap} Seperent's Offerings lost to overcapping.
+				{this.snilkGauge.overCap} Serpent's Offerings lost to overcapping.
```

Comment:
super nitpicky but I think "Serpent's Offerings" should be "Serpent Offerings".
 
 > The use of certain weaponskills in the execution of combos will serve to fill the Serpent Offerings Gauge.